templates/contrib/pc/README.md

```diff
@@ -0,0 +1,23 @@
+# Overview
+
+This directory provides a RHEL6 based template, to support legacy guest OS
+which require an older pc-i440fx machine type to boot.
+Clusters that were deployed using the HyperConverged Operator do not include
+support for this machine type. To allow Virtual Machines to request this
+machine type, this option should be manually added.
+
+# Update HCO
+
+It is necessary to add the relevant machine type to a list of permitted types.
+This will allow Virtual Machines to request the pc-i440fx machine type.
+
+$ oc annotate --overwrite -n openshift-cnv hco kubevirt-hyperconverged \
+  kubevirt.kubevirt.io/jsonpatch='[{"op": "add", \
+    "path": "/spec/configuration/emulatedMachines", \
+    "value": ["q35*", "pc-q35*", "pc*"] }]'
+
```

Comment:
Is the updated annotation applied instantly, or is an additional step required?

Reply:
Does this expression also need ot be extended to permit i.e. a RHEL machine type?

Reply:
> Does this expression also need ot be extended to permit i.e. a RHEL machine type?

No, this doesn't need to be as rhel is a suffix. Unless we want to fixate on a specific one (pc-i440fx-rhel7.6.0), however, in that case, a slight change will lead to changes in the template.

Reply:
> Is the updated annotation applied instantly, or is an additional step required?

As far as I know, no additional steps are required.

Reply:
On second thought, I'll change this to match the template.

Reply:
Just to double check... @nunnatsa would any additional steps be required here to change the `emulatedMachines` value in KubeVirt CR via the HCO CR?

Reply:
@nunnatsa ?

Reply:
I modified the machine-type of kubevirt-config ConfigMap to let kubevirt accept the new machine type.

Reply:
I this is a formally required solution, I think it will be better to add it to HCO API. The annotation is for workarounds, solving urgent issues, and so on.

Maybe it will be better to apply a PR in HCO with a suggestion for a new API?

Reply:
> Maybe it will be better to apply a PR in HCO with a suggestion for a new API?

Adding non-default machine types will stay an exception. For this reason, it might not worth an API change.

It might be helpful for the users, if we would explain in the documentation, how allow the new machine type not only with the upcoming release by the annotation, but also for the latest release 2.6 using the config map.

Reply:
> It might be helpful for the users, if we would explain in the documentation, how allow the new machine type not only with the upcoming release by the annotation, but also for the latest release 2.6 using the config map.

This is a little bit tricky, as the config will override any KubeVirt CR configuration. Therefore, all the configuration should be done either on the CR or on the config map.

@nunnatsa does the annotation work in the 2.6 release as well?

Reply:
> @nunnatsa does the annotation work in the 2.6 release as well?

No, it's not.

Reply:
OK, thanks. So what is the best approach to update this in 2.6? Should we suggest a config map or just update KubeVirt CR directly?